styles/style.css

```diff
@@ -1,18 +1,239 @@
-/* header{
+
+header{
     background-color: #1a1a1a;
 }
 body {
     color: #333;
     background-color: #f9f9f9;
     font-family: 'Arial', sans-serif;
     font-size: 1rem;
+    margin :0;
+    padding: 0;
+    width: 100%;
+    height: 100%;
 }
+
+h2{
+    text-align: center;
+   
+}
+a{
+    text-decoration: none;
+    color:white;
+    padding: 1rem;
+}
+
 nav{
-    
             color: white;
             text-align: center;
-            padding: 1em 0;
+            
 }
 footer{
+  text-align: center;
     background-color: #1a1a1a;
-} */
\ No newline at end of file
+}
+
+/* classes  */
+
+.choose_us_heading{
+    margin-top: 1rem;
+}
+
+.header_tabs{
+padding: 1rem;
+
+}
+
+.choose_us{
+    display: flex;
+    margin-bottom: 3rem;
+    flex-wrap: wrap;
+
+    
+}
+
+.what_we_do{
+    background-color: #f1f1f1;
+  padding-top: 1.5rem;
+  padding-left: 2.5rem;
+  padding-right: 2.5rem;
+  padding-bottom: 2.5rem;
+ 
+}
+.boxes_2{
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 10px;
+   
+}
+.boxes_3{
+    display: grid;
+    grid-template-columns: repeat(4,1fr);
+    gap: 30px;
+    padding-left: 1.5rem;
+    padding-right: 1.5rem;
+
+}
+.trending_news{
+    margin-top: 2.5rem;
+   
+    
+
+}
+.trending_new_heading{
+    margin-bottom: 2rem;
+}
+.footer_data{
+ color:white;
+  
+}
+.footer_nav{
+    padding-bottom: 1rem;
+    padding-top: 0.5rem;
+}
+
+.privacy{
+    padding: 1rem;
+}
+.terms{
+    padding: 1rem;
+}
+.contact_div{
+    margin-top: 4rem;
+    height: 100px;
+}
+/* id  */
+
+#choose_box h3{
+    text-align: center;
+
+
+}
+#choose_box {
+    width: 40%;
+    height: 120px;
+    margin: auto;
+    background-color: #fff;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+    border: 1px solid #ddd;
+    padding-left: 1rem;
+    padding-bottom: 0.5rem;
+    padding-right: 1rem;
+    text-align: center;
+}
+#nav_link{
+    line-height: 1.8;
+}
+
+#box_2{
+
+    width: auto;
+    margin: 1.2rem;
+    min-height: 170px;
+    background-color: #fff;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+    border: 1px solid #ddd;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    text-align: left;
+}
+#box_3{
+    min-width: 130px;
+    
+    min-height: 160px;
+    background-color: #fff;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+    border: 1px solid #ddd;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    text-align: center;
+}
+
+@media only screen and (max-width: 775px) {   /*upto 800px*/
```

Comment:
To avoid conflicts in different type of Tablet view , use min-width and max-width both for consistent structure for elements.
EX. (min-width:320px) and (max-width:768px)